pkgs/data/themes/cyberpunk-neon/default.nix

```
@@ -0,0 +1,39 @@
+{ lib, stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+    pname = "cyberpunk-neon";
+    version = "b4b293c";
```

```suggestion
    version = "2021-02-02";
```
https://nixos.org/manual/nixpkgs/stable/#sec-package-naming